lib/mapper.js

```diff
@@ -8,6 +8,36 @@ var unixTime = require('unix-time');
 var objCase = require('obj-case');
 
 /**
+ * Turn a JSON product into Iterable format
+ * Specifically, any non-special fields should be nested under a field called "dataFields"
+ *
+ * @param cart
+ * @returns {Array}
+ */
+function cartToIterableCart(cart) {
+  var commerceItemFields = ["id", "sku", "name", "description", "categories", "price", "quantity", "imageUrl", "url"];
+  var numCommerceItemFields = commerceItemFields.length;
+  var items = [];
+  var i, len = cart.length;
+  for (i = 0; i < len; i++) {
```

Comment:
we tend to shy away from loops. prefer array methods and helpers like `@ndhoule/foldl` to build up new objects comprising only the properties we need w/o side effects.